srtcore/group.cpp

```diff
@@ -2372,21 +2372,20 @@ int CUDTGroup::recv(char* buf, int len, SRT_MSGCTRL& w_mc)
                 // by "more correct data" if found more appropriate later. But we have to
                 // copy these data anyway anywhere, even if they need to fall on the floor later.
                 int stat;
+                char lostbuf[SRT_LIVE_MAX_PLSIZE];
+                char* msgbuf = NULL;
                 if (output_size)
                 {
                     // We have already the data, so this must fall on the floor
-                    char lostbuf[SRT_LIVE_MAX_PLSIZE];
+                    msgbuf = lostbuf;
```

Comment:
I think you better fix the explanation here as it's no longer true. The `lostbuf` isn't also the best name now, as this is now a "maybe lost buf". Maybe `sparebuf` would be more appropriate, and the comment above should not speak about "falling on the floor", but rather sound like "most likely this is an already received packet, but you never know what packet it is until you read it. In case when it turns out to be an ahead packet received by having the expected current packet being TLPKTDROP-ed from this link, this packet will have to be saved as ahead packet instead."

Reply:
Renamed to `extrabuf`.